Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
+   humanChoice = this.id;
+
+	 var choices = ["rock", "paper", "scissors"];
+
+	 var computerChoice = choices[Math.floor(Math.random() * choices.length)];
+
+   compare(humanChoice, computerChoice);
+
+	 document.querySelector("#computerScore").innerHTML = computerScore;
+	 document.querySelector("#humanScore").innerHTML = humanScore;
+
+
+});
+
+
+
+function compare (humanChoice, computerChoice){
+  if (humanChoice === computerChoice) {
+
+    document.querySelector("#status").innerHTML = "It's a tie!";
+		}
+
+  	else if (humanChoice === "rock") {
+
+
+ 		if (computerChoice === "scissors") {
+
+    humanWins();
+
+		} else {
+
+		botWins();
+		}
+	}
+
+
+
+		else if (humanChoice === "paper") {
+
+
+		 if (computerChoice === "rock") {
+
+			 humanWins();
+
+		} else {
+
+			botWins();
+		  }
+		}
+
+
+
+
+		 else if (humanChoice === "scissors") {
+
+
+		 if (computerChoice === "rock") {
+
+			 botWins();
+		} else {
+
+			humanWins();
+		}
+	}
+};
+
+function humanWins(){
```

Comment:
I like that you've created these two functions to isolate the effects of the human or bot winning, good call! 👍